Apeswap-Bots/detect-large-banana-token-mints/README.md

```diff
@@ -0,0 +1,30 @@
+# Large BANANA Token Mint
+
+## Description
+
+This bot detects transactions with large BANANA token mints
+
+## Supported Chains
+
+- Binance Smart Chain
+
```

Comment:
Polygon should be added here.

Also, please mention in the Readme that the threshold can be configured too and where.